blog/_posts/2019-01-18-fluxdiffeq.md

```diff
@@ -64,15 +64,18 @@ if you know the exact functional form that relates the input to the output.
 However, in many cases, such exact relations are not known *a priori*.
 So how do you do nonlinear modeling if you don't know the nonlinearity?
 
-There are two separate ways to then handle this problem. One method is machine
-learning. Usually this is presented as you have some $x$ and you want to predict
-$y$. This generation of a prediction from $x$ is the machine learning model, call it
-$ML$, where you put $x$ into the program and it spits out values $y$. But if you think
+There are two common ways to handle this problem. One method is to use machine

```

Comment:
You still need to tell me both of them immediately.
Then you can go into details abotu each.